static-default/party/overlay.html

```diff
@@ -41,4 +61,19 @@
   });
 });
 
+Service.get('chat', chat => {
+  chat.observe('message', message => {
+    const li = document.createElement('li');
+    li.innerText = message;
+
+    messages.appendChild(li);
+  });
+
+  newMessage.addEventListener('submit', (event) => {
+    event.preventDefault();
+    chat.addMessage(newMessageInput.value);
```

Comment:
Very optional: you could give the input a name like "message" and then access it as `chat.addMessage(newMessage.message.value)`, instead of using the id.